routes/markers.js

```diff
@@ -249,6 +251,7 @@ module.exports = function(connection) {
                         else {
                             res.setHeader('Content-Type', 'application/json');                    
                             res.json(data);        
+                            utils.addRSS(connection, 'Marker Updated', `${data.name} updated on OpenMRS Atlas`, req.session.user.uid);
```

Comment:
* Title: ${data.name} entry updated
* Description: ${username} made changes to ${data.name} on the OpenMRS Atlas